inst/@sym/subsasgn.m

```diff
@@ -517,4 +518,71 @@
 %! a = sym([1, 2; 3, 4]);
 %! a(1, 2) = [];
 
+
+%% Tests from mat_replace
```

Comment:
I have added this as al the tests here on after mostly relate to the changes done in mat_replace only.